### PR TITLE
Update manual workflow to be compatible with bake changes

### LINKED ISSUE #748: Fix manual build pipeline for compatibility with `bake`
The [manual pipeline](./.github/workflows/build-manual.yaml) wasn't updated in conjunction with the changes made to adopt `bake` for image building. We should update it to be compatible with `bake` so it can be used by relevant product teams in one-off builds.

### PR DESCRIPTION
**Pull request by ianpittwood**
Fixes #748

### CODE REVIEW

**Comment by ianpittwood** [comment]
Testing this out with a Package Manager preview: https://github.com/rstudio/rstudio-docker-products/actions/runs/8972259024/job/24639843912

File: .github/workflows/build-bake-preview.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 on:
   schedule:
-    - cron: '0 * * * *'
+    - cron: '0 8 * * *'
```
Comment (tylfin) [inline .github/workflows/build-bake-preview.yaml:3]: Do we have to revert this now or can we fix forward https://github.com/rstudio/rstudio-docker-products/issues/749

Comment (ianpittwood) [reply .github/workflows/build-bake-preview.yaml:3]: I don't have a plan on how to implement your request yet. Does PPM still need builds that frequent even after the release?

Comment (ianpittwood) [reply .github/workflows/build-bake-preview.yaml:3]: I'll revert this for now so we can keep moving forward with other work.

**Review by bschwedler — APPROVED** [review]

**Pull request merged**